includes/Modules/Analytics/Account_Ticket.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Provisioning
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics;
+
+use Google\Site_Kit_Dependencies\Google_Service_Analytics_AccountTicket;
+
+/**
+ * Class for the Analytics provisioning Account Ticket object.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class AccountTicket extends Google_Service_Analytics_AccountTicket {
```

Comment:
Just an idea, but maybe call this something like `Proxy_AccountTicket` or similar to further clarify the purpose?

Reply:
yea, was wondering about that as well, good suggestion. renamed